src/picongpu/include/traits/PICToAdios.tpp

```diff
@@ -32,7 +34,7 @@ namespace traits
 {
     /** Trait for int */
     template<>
-    struct PICToAdios<int>
```

Comment:
If we remove this with fixed size types than we should add an description at the begin of the files that it is not allowed to define native C/C++ includes types e.g., `unsigned long`, `int` ,... because they can conflict with the types `int32_t`, `uint64_t`, ...